### DISCUSSION TITLE
Ability to change default values for each instance separately in Table field

### DISCUSSION THREAD

**Discussion by foxtrotcz** [craftcms/cms#15312] [Ideas] [answered]
Hi,
this is kinda extension of my previous feature request: https://github.com/craftcms/cms/discussions/9028

Currently I can set static default values for Table field which show in every instance and change everywhere if I change them in field definition. That's great.

But I discovered that now with instances I would like to set these static default values differently for every instance without being forced to create new Fields in Field editor.

Lets say I have Table 2Col field which is just basic Table with 2 columns and without any default values set.
Now I want to use this Table in multiple Entry Types or multiple times in one Entry Type.
I would just modify that one instance similar to how I can rename each instance and change handle for each instance.
For address instance:
![Bez názvu](https://github.com/craftcms/cms/assets/10333810/7e0e11c2-5c6e-45ca-aa55-44607b040150)

For contact info instance:
![Bez názvu](https://github.com/craftcms/cms/assets/10333810/4bb57f6f-222d-4a9a-bc2b-3395a887d6a8)

This way I could customize each instance of the Table field without needing to create new fields. This is useful when you want this specific configuration only one time in one instance. It's easier than creating new fields for each one-time instance.

In entries it will look like this. User sees basically form they will fill out. They cannot edit the default values in 1st column.
![image](https://github.com/craftcms/cms/assets/10333810/6b808481-0267-47d5-8730-314cd9e2080e)

Thanks for consideration.

**Comment by brandonkelly** [comment, marked as answer]
Thanks for the suggestion. The basic idea that field settings should be overridable from field layouts is already being discussed in #8688.
